### PR TITLE
Update `make install-watch` and `install.sh` to clean up symlinks

### PR DESCRIPTION
**Pull request by umar-tophat**
## Description

- Rather than link the entire `artifacts/webpack_build` folder to `~/.yvm`, I decided to link the files that are actually changing (ie. `yvm.js` and `yvm.sh`). 
- Reasoning behind this that subsequent install commands will install into `artifacts/webpack_build` and removing the link will remove the versions you installed.

### DevQA Steps

- Run `make install-watch`
	- Verify that changes to code are updated without having to re-run install
- Stop watcher, run `make install`
	- Verify that links are removed, and normal install is completed

### CODE REVIEW

File: scripts/install-watch.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+rm ${HOME}/.yvm/yvm.sh
+rm ${HOME}/.yvm/yvm.js
+
+# Link
+ln -s ${CURRENT_DIR}/artifacts/webpack_build/yvm.sh ${HOME}/.yvm/yvm.sh
+ln -s ${CURRENT_DIR}/artifacts/webpack_build/yvm.js ${HOME}/.yvm/yvm.js
+chmod +x ${HOME}/.yvm/yvm.sh
+
+# Start Webpack in watch mode
+webpack --progress --config webpack/webpack.config.dev.js --watch
```
Comment (francoiscampbell) [inline scripts/install-watch.sh]: how is the previous yvm recovered? maybe we should backup the current files and restore them after the `webpack` returns?

Comment (umar-tophat) [reply scripts/install-watch.sh]: Issue is how to execute command after Webpack. If you send Interrupt signal, the lines of the script that come after are not executed.

Comment (francoiscampbell) [reply scripts/install-watch.sh]: you can trap the interrupt and clean up: https://stackoverflow.com/questions/22752461/continuing-a-script-after-ctrl-c

File: scripts/install-watch.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+# Link
+ln -s ${CURRENT_DIR}/artifacts/webpack_build/yvm.sh "${HOME}/.yvm/yvm.sh"
+ln -s ${CURRENT_DIR}/artifacts/webpack_build/yvm.js "${HOME}/.yvm/yvm.js"
+chmod +x "${HOME}/.yvm/yvm.sh"
+
+# Trap interrupt signal
+int_trap() {
+    echo "Stopped Webpack"
+}
+trap int_trap INT
```
Comment (jakebolam) [inline scripts/install-watch.sh]: woah

File: scripts/install-watch.sh [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+# Start Webpack in watch mode
+webpack --progress --config webpack/webpack.config.dev.js --watch
```
Comment (jakebolam) [inline scripts/install-watch.sh]: This works just like this? Sweet

**Comment by jakebolam** [comment]
Fixes #77

**Pull request merged**